modules/seqexec/model/src/main/scala/seqexec/model/enum/ApplyCommandResult.scala

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.model.enum
+
+import gem.util.Enumerated
+
+sealed trait ApplyCommandResult extends Product with Serializable
```

Comment:
This was called `EpicsCommand.Result`. It is confusing that we have at least 3 classes called `Result` and in fact these are not used only in Epics code. Hence I thought about moving them to the model